File: LeetCode-challenges/TwoSum/twosum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        dicth = {}
        list1 = []
        for i, num in enumerate(nums):
            complement = target-num
            if complement in dicth:
                list1.append(i)
                list1.append(dicth[complement])
                return list1
            dicth[num] = i

File: LeetCode-challenges/TwoSum/twosum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    let map = {};
    let list1 = []
    for(let i = 0;i<nums.length;i++){
        let complement = target-nums[i]
        if(complement in map){
            list1.push(i)
            list1.push(map[complement])
            return list1
        }
        map[nums[i]] = i;
    }
};